### ISSUE TITLE
`test_rhash.sh` fails test #8 when executed in bash

### ISSUE THREAD

**Issue by lvd2** [rhash/RHash#221]
In Debian and Ubuntu, `/bin/sh` is mapped to dash, and in those systems `test_rhash.sh` completes all tests successfully.
In systems where `/bin/sh` is mapped to bash, or when executing test script specifically in bash, test #8 fails as follows:
```
$ bash --version
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
Copyright (C) 2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
$ bash test_rhash.sh ../rhash
...
 8. test special characters:    Failed Subtest #3
obtained: "(stdin):1:can'tparseline"\\00000000a/\\r1""
expected: "\a/\r1OK"
Failed Subtest #4
obtained: "(stdin):1:can'tparseline"\\00000000a/\\n2""
expected: "\a/\n2OK"
Ok
```

**Comment by rhash** [comment]
Thanks for reporting the bug!

Here is the fix: 1486a21fd72b884ed9edd61e050f5e2dae93e29a.

**Comment by rhash** [comment]
Fixed in RHash v1.4.4.
